tests/reals/real_constbuffer_array_tarray.c

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include "real_gballoc_hl.h"
+#include "real_gballoc_hl_renames.h"
+#include "c_pal/gballoc_hl_redirect.h"
```

Comment:
>clude "c_pal/gballoc_hl_redirect.h" [](http://example.com/codeflow?start=3&length=35)

Is this needed?

Reply:
removed

Reply:
added back, it's needed in THANDLE_LL_TYPE_DEFINE